django_modern_rest/types.py

```diff
@@ -1,10 +1,26 @@
 import dataclasses
 from collections.abc import Callable
-from typing import Any, Final, final, get_args, get_origin
-
+from typing import Any, Final, final, get_args, get_origin, TypedDict
 from typing_extensions import get_original_bases, get_type_hints
 
 from django_modern_rest.exceptions import UnsolvableAnnotationsError
+from django_modern_rest.openapi.config import OpenAPIConfig
+
+
+#  TypedDict for Django Modern REST settings
+
+
+class DMRSettings(TypedDict, total=False):
+    """
+    TypedDict defining the shape of django-modern-rest settings.
+    """
+
+    serialize: str | Callable[..., Any]
```

Comment:
Please, use correct types for `serialize` and `deserialize`. See `internal/json`